### PR TITLE
Updated photography

### PR DESCRIPTION
**Pull request by mike-yesware**
Added concent notice and changed the photo archiving instructions

### CODE REVIEW

File: task-descriptions.md [hunk 1]
```diff
@@ -56,9 +56,11 @@ Keep an eye on time - allow for flexibility, but also make sure we're moving alo
 
 Time commitment: part or all of the workshop; selecting && emailing photos afterward
 
+* Always ensure concent fron your subjects.
 * Take brilliant pictures.
-* Post them to twitter?
-* Send them to a workshop coordinator to post on the RailsBridge Boston website or on Twitter.
+* Post them to twitter!
```
Comment (edwardloveall) [inline task-descriptions.md]: ❗️

File: task-descriptions.md [hunk 2]
```diff
@@ -56,9 +56,11 @@ Keep an eye on time - allow for flexibility, but also make sure we're moving alo
...
+* Post them to a folder for the event in G Drive: https://drive.google.com/drive/u/0/folders/1nvGBaLCOxtRo8zRWnH3EqYd91-VODYlR
```
Comment (edwardloveall) [inline task-descriptions.md]: What do you think about making this a private folder somehow? Even if people give consent it feels weird to me to put their photos online in a public place that they may not know about.

Comment (raadler) [reply task-descriptions.md]: What if when we send out the welcome e-mail we mention our twitter and that event photos of y ou will end up there if you consent to being photoed?

Comment (mike-yesware) [reply task-descriptions.md]: I'm fine with that, but I will say that all these photos have been on Twitter. Might be nice for the folder to contain ones we don't tweet though.

Done.

File: task-descriptions.md [hunk 3]
```diff
@@ -56,9 +56,11 @@ Keep an eye on time - allow for flexibility, but also make sure we're moving alo
 
 Time commitment: part or all of the workshop; selecting && emailing photos afterward
 
+* Always ensure concent fron your subjects.
```
Comment (amandabeiner) [inline task-descriptions.md]: Minor: consent

[the remark this answers is not in the document]
Comment (mike-yesware) [reply task-descriptions.md]: Actually, there isn't always a single photographer, so I added notes to this effect in the MC section instead. What do you think @amandabeiner?

**Comment by mike-yesware** [comment]
I think I’ve addressed all the feedback.

**Pull request merged**